postgresql/installingpostgresqlonibmi.md

```diff
@@ -0,0 +1,167 @@
+# Install and configure Postgresql DB Server on IBM i 
+
+Install postgres yum packages from IBM i ACS Open Source Package Management
+```
+Package list:
+postgresql12 ***(This item may be the only one you need to select to include all packages)***
+postgresql12-contrib
+postgresql12-devel
+postgresql12-docs
+postgresql12-libecpg
+postgresql12-libogtypes
+postgresql12-libpq
+postgresql12-server
+```
+
+Create IBM i user: ***POSTGRES*** and give it a password. It's important the user is named POSTGRES because the Postgresql server needs a default user of POSTGRES. At least that was my experience with it. (Also give it *ALLOBJ access for now. You can probably remove later and possibly even disable the IBM i POSTGRES user. ***Not tested.***)
+
+Log in to SSH as IBM i user ***POSTGRES*** to do server setup.
+
+If not started, start the bash shell by typing ***bash*** unless bash is already your default shell. 
+
+Run the following shell command line sequence to initialize postgres database in the /postgres IFS directory location
+```
+cd /
+
+mkdir postgres
+
+export PGDATA=/postgres
+
+initdb -E UTF-8 -D /postgres
```

Comment:
By exporting `PGDATA` you do not have to specify `-D /postgres`

![image](https://user-images.githubusercontent.com/33973272/83290801-df3d3a00-a1ac-11ea-9f22-98500521976d.png)

https://www.postgresql.org/docs/current/app-initdb.html

Reply:
I would suggest using the command line option over the environment variable as its more explicit and clearer.

Reply:
Let’s leave both so there’s no confusion.

Reply:
That sounds _more_ confusing to me.

Reply:
You could just make the edits you want yourself.  As long as the user types what we said it should be just fine.